test/studies/ml/lib/Chai.chpl

```diff
@@ -0,0 +1,828 @@
+
+module Chai {
+    
```

Comment:
Could you add a block comment up here that summarizes the interface for a layer. It could be a bulleted list with each of the methods that the layers support, with a brief description of what they each do.